server/routes.go

```diff
@@ -50,8 +51,10 @@ func DoRoutes() chi.Router {
 	// and return a http.Handler.  This is normally used with .Mount,
 	// but since only part of the server is using code gen this is
 	// a way to hack it in
-	seatManagerApi := controllers.NewSeatManagerApi(amsClient, bopClient)
-	api.HandlerFromMuxWithBaseURL(seatManagerApi, r.With(identity.EnforceIdentity), "/api/entitlements/v1")
+	if !configOptions.GetBool(config.Keys.DisableSeatManager) {
```

Comment:
I think this is a little confusing to read but I wanted to name the env var in a way to imply this is enabled by default, and to convey that a lack of this variable means its enabled. I can switch if anyone prefers